app/models/item_order.rb

```diff
@@ -4,7 +4,7 @@ class ItemOrder <ApplicationRecord
   belongs_to :item
   belongs_to :order
 
-  enum status: %w(pending unfulfilled)
+  enum status: %w(pending unfulfilled fulfilled)
```

Comment:
Maybe eventually it'd make sense to change this from an enum to a boolean (i.e., `fullfilled?, default: false`). Is an `item_order` ever pending? Or is it just `unfulfilled` or `fulfilled`?

Reply:
I'm not sure what they want the default to be, because in User Story #30:
When I click the cancel button for an order, the following happens:
- Each row in the "order items" table is given a status of "unfulfilled"

Since it is given the "unfulfilled" status at that point, it seems like maybe it shouldn't be the default?

Reply:
Yeah... I see what you mean, US 30 is sort of a confusing one. Probably okay to keep them as enums for now...